app/src/main/java/com/mapbox/android/gestures/testapp/MapboxActivity.java

```diff
@@ -0,0 +1,83 @@
+package com.mapbox.android.gestures.testapp;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.text.TextUtils;
+
+import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.maps.MapView;
+
+import timber.log.Timber;
+
+/**
+ * Test activity showcasing a simple MapView with current mapbox-gestures-android library commit.
+ */
+public class MapboxActivity extends AppCompatActivity {
+
+  private static final String DEFAULT_MAPBOX_ACCESS_TOKEN = "YOUR_MAPBOX_ACCESS_TOKEN_GOES_HERE";
+  private static final String ACCESS_TOKEN_NOT_SET_MESSAGE =
```

Comment:
this error will also be thrown by the Mapbox.java if you don't set a token

Reply:
`Mapbox.java` actually throws [a bit different one](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/exceptions/MapboxConfigurationException.java). This additionally includes a hint about `developer-config.xml`.